Sources/ArcGISToolkit/Components/Offline/PreplannedMapModel.swift

```diff
@@ -318,7 +320,8 @@ extension PreplannedMapArea: PreplannedMapAreaProtocol {
     }
     
     var description: String {
-        portalItem.description
+        // Remove HTML tags from description.
+        portalItem.description.replacingOccurrences(of: "<[^>]+>", with: "", options: .regularExpression)
```

Comment:
Please use a Swift regex expression instead.

```suggestion
        portalItem.description.replacing(/<[^>]+>/, with: "")
```

Note: This suggestion will requiring enabling the `BareSlashRegexLiterals` upcoming feature:

```swift
.enableUpcomingFeature("BareSlashRegexLiterals")
```

Which can then be removed as part of the work to move to Swift 6 as that feature is enabled by default.